tetrateci/create_istio_release.sh

```diff
@@ -93,8 +93,8 @@ if [ ${TAG} =~ "fips" ]; then
 fi
 
 go run main.go publish --release /tmp/istio-release/out --dockerhub $HUB
-echo "Cleaning up the docker build...."
-[ -d "/tmp/istio-release" ] && sudo rm -rf /tmp/istio-release
+echo "Cleaning up the istio source artificats...."
+sudo rm -rf /tmp/istio-release/sources/
```

Comment:
As far as I understand, this PR moves the original `sudo rm -rf /tmp/istio-release` to the bottom of this script.

Why do we need to add `sudo rm -rf /tmp/istio-release/sources/` here?

Reply:
when the create_istio script executes go run main.go build --manifest manifest.archive.yaml, it has dependency on docker folder built in previous go build steps. We need to remove istio-release/sources because it again create/clone source folder in /tmp/istio-release and complains if this is not empty